src/main/resources/hudson/plugins/jira/listissuesparameter/JiraIssueParameterDefinition/index.jelly

```diff
@@ -17,25 +17,10 @@ limitations under the License.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
   <j:set var="escapeEntryTitleAndDescription" value="false"/>
-  <f:entry title="${h.escape(it.name)}" description="${it.formattedDescription}">
-    <!-- this div is required because of ParametersDefinitionProperty.java#117 -->
-    <div name="parameter">
+  <j:set var="instance" value="${it}"/>
+  <j:set var="descriptor" value="${it.descriptor}"/>
+  <f:entry field="value" title="${h.escape(it.name)}" description="${it.formattedDescription}">
       <input type="hidden" name="name" value="${it.name}"/>
-      <j:choose>
-        <j:when test="${it.issues == null or it.issues.size() == 0}">
```

Comment:
I did`t have now direct access to the list of items. Then now coming form the fill method.
How should this now be handled without fetching the issues again?

Reply:
Can you keep the logic for no-items found?

Reply:
Maybe it also nice to have a parameter to say if the parameter is required. 
Then we could prevent pressing the build button without a valid issue selected.
Ex the git-parameter has it.
Something for a new PR.